### ISSUE TITLE
Feature Request/Idea: Display the number of active users at the moment

### ISSUE THREAD

**Issue by sergejzr** [IQSS/dataverse#10292]
**Overview of the Feature Request**
Dear dataverse developers,

It is a common feature in phpBB to display  statistics information about currently active users. For example:
![image](https://github.com/IQSS/dataverse/assets/12542792/432796ac-5bad-4dc9-94d7-84000efe0cd8)
![image](https://github.com/IQSS/dataverse/assets/12542792/1a5fcf0f-11fb-4fad-a9f1-c27db27b1de6)
 Similar feature would be good, at least to show at admin dashboart.

**What kind of user is the feature intended for?**
(Example users roles: API User, Curator, Depositor, Guest, Superuser, Sysadmin)
At least admins, but could be shown to all users too.

**What inspired the request?**
sometimes for small maintainance we would like to know, how many users are active at the moment. 
Also this information could be interesting for guestbook users.

**Any open or closed issues related to this feature request?**
Did not find any...

**Comment by pdurbin** [comment]
@sergejzr good idea. Thanks. Have you considered using Google Analytics or similar for this? (I'm not sure if Matomo supports it.) That's where I've seen a dashboard of how many active users there are.

**Comment by sergejzr** [comment]
Thanks Phil,
google analytics is not really an option for the Uni, but we have matomo, I will see what is possible..

**Comment by cmbz** [comment]
2025-06-30
- @jggautier will follow up with poster about current status for their installation and possible broader appeal for other site visitors

**Comment by jggautier** [comment]
Hey @sergejzr. As far as I can tell, matomo does let people who have access to it see the number of active users. Just did a quick Google search and found https://matomo.org/faq/how-to/faq_189.

Could you let us know if as an admin, matomo shows you the count of active users in the repository you manage?

If so, and if you're still interested in everyone who visits the repository being able to see the number of active users, could you create a new GitHub issue about this?

**Comment by jggautier** [comment]
I'm going to close this GitHub issue as not planned. @sergejzr please feel free to reply and open it if you're still interested
